OpenEdXMobile/src/main/java/org/edx/mobile/view/TabsBaseFragment.java

```diff
@@ -91,16 +92,23 @@ private void handleTabSelection(@Nullable Bundle bundle) {
         }
     }
 
-    public boolean shouldSelectFragment(@NonNull FragmentItemModel item, @NonNull String screenName) {
+    /**
+     * Determines if a tab fragment needs to be selected based on screen name.
+     *
+     * @param item {@link FragmentItemModel} assigned to a tab.
+     * @param screenName screen name param coming from {@link DeepLinkManager}
+     * @return <code>true</code> if the specified tab fragment needs to be selected, <code>false</code> otherwise
+     */
+    private boolean shouldSelectFragment(@NonNull FragmentItemModel item, @NonNull String screenName) {
         return
             (screenName.equals(Screen.PROGRAM) && item.getIcon() == FontAwesomeIcons.fa_clone) ||
             (screenName.equals(Screen.COURSE_DISCOVERY) && item.getIcon() == FontAwesomeIcons.fa_search) ||
             (screenName.equals(Screen.PROGRAM_DISCOVERY) && item.getIcon() == FontAwesomeIcons.fa_search) ||
             (screenName.equals(Screen.COURSE_VIDEOS) && item.getIcon() == FontAwesomeIcons.fa_film) ||
             (screenName.equals(Screen.COURSE_DISCUSSION) && item.getIcon() == FontAwesomeIcons.fa_comments_o) ||
             (screenName.equals(Screen.COURSE_DATES) && item.getIcon() == FontAwesomeIcons.fa_calendar) ||
-            (screenName.equals(Screen.COURSE_HANDOUTS) && item.getIcon() == FontAwesomeIcons.fa_ellipsis_h) ||
-            (screenName.equals(Screen.COURSE_ANNOUNCEMENTS) && item.getIcon() == FontAwesomeIcons.fa_ellipsis_h)
+            (screenName.equals(Screen.COURSE_HANDOUT) && item.getIcon() == FontAwesomeIcons.fa_ellipsis_h) ||
+            (screenName.equals(Screen.COURSE_ANNOUNCEMENT) && item.getIcon() == FontAwesomeIcons.fa_ellipsis_h)
             ;
     }
```

Comment:
This whole function with its Javadoc can be reformatted as follows:

```java
    /**
     * Determines if a tab fragment needs to be selected based on screen name.
     *
     * @param item       {@link FragmentItemModel} assigned to a tab.
     * @param screenName screen name param coming from {@link DeepLinkManager}
     * @return <code>true</code> if the specified tab fragment needs to be selected, <code>false</code> otherwise
     */
    private boolean shouldSelectFragment(@NonNull FragmentItemModel item, @NonNull String screenName) {
        return (screenName.equals(Screen.PROGRAM) && item.getIcon() == FontAwesomeIcons.fa_clone) ||
                (screenName.equals(Screen.COURSE_DISCOVERY) && item.getIcon() == FontAwesomeIcons.fa_search) ||
                (screenName.equals(Screen.PROGRAM_DISCOVERY) && item.getIcon() == FontAwesomeIcons.fa_search) ||
                (screenName.equals(Screen.COURSE_VIDEOS) && item.getIcon() == FontAwesomeIcons.fa_film) ||
                (screenName.equals(Screen.COURSE_DISCUSSION) && item.getIcon() == FontAwesomeIcons.fa_comments_o) ||
                (screenName.equals(Screen.COURSE_DATES) && item.getIcon() == FontAwesomeIcons.fa_calendar) ||
                (screenName.equals(Screen.COURSE_HANDOUT) && item.getIcon() == FontAwesomeIcons.fa_ellipsis_h) ||
                (screenName.equals(Screen.COURSE_ANNOUNCEMENT) && item.getIcon() == FontAwesomeIcons.fa_ellipsis_h);
    }
```